README.md

```diff
@@ -5,6 +5,17 @@ Tools to capture Vulkan/OpenGL applications with OBS. For instructions on how to
 
 For this to work you also need the corresponding OBS VkCapture plugin for OBS Studio. This plugin is available as [`com.obsproject.Studio.Plugin.OBSVkCapture` on Flathub](https://github.com/flathub/com.obsproject.Studio.Plugin.OBSVkCapture).
 
+Installation
+-----------
+
+This plugin is an extension for Freedesktop Platform therefore is not listed on the Flathub website to download it.
```

Comment:
Same as https://github.com/flathub/com.obsproject.Studio.Plugin.OBSVkCapture/pull/17

Should be
> This flatpak is an extension for the Freedesktop platform and is therefore not listed on the Flathub website.

Reply:
Text replaced in the last commit with the one you provided.